third_party/llm_engine/llm_engine.bzl

```diff
@@ -19,8 +19,8 @@ def llm_engine():
     llm_engine_repository(name="_llm_engine")
     new_git_repository(
         name = "llm_engine",
-        remote = "https://github.com/mzegla/openvino.genai",
-        commit = "582af04ae297a5ebc8246e326979495ab2e12748",
+        remote = "https://github.com/rasapala/openvino.genai",
```

Comment:
Is it supposed to be different fork or is it just for test?

Reply:
Need to merge a PR with review from Ilya to switch back to his repository.

Reply:
PR merged, let's switch to the main fork:
https://github.com/ilya-lavrenov/openvino.genai/commits/ct-beam-search/